guides/get-started-microservices-on-kubernetes/simple-microservice-infinispan-part1.adoc

```diff
@@ -0,0 +1,286 @@
+
+[[prerequisites]]
+== Prerequisites
+
+To complete this guide, you need:
+
+* Complete link:simple-microservice-part1[{simple-microservice-part1}]
+
+include::_includes/_constants.adoc[]
+
+== Infinispan Server
+
+Infinispan is the most common solution used with WildFly when you want to cache some data outside WildFly.
+
+Using Infinispan, for example, you can avoid users loosing their session in case WildFly is shut down and re-started.
+
+In this post, we will use the containerized version of Infinispan: see link:https://quay.io/repository/infinispan/server?tab=tags&tag=latest[Infinispan Server, window="_blank"].
+
+=== Start Infinispan [[start_infinispan_container]]
+
+[source,bash,subs="normal"]
+----
+podman network create {podman-network-name}
+
+podman run --rm --network={podman-network-name} --name={infinispan-pod-name} \
+  -p {infinispan-port}:{infinispan-port} \
+  -e USER="{infinispan-user}" \
+  -e PASS="{infinispan-password}" \
+  {infinispan-server-docker-image}
+----
+
+NOTE: if you are using podman on Linux, then run the previous commands using link:https://en.wikipedia.org/wiki/Sudo[sudo]
+
+NOTE: NOTE: We created the *{podman-network-name}* network and started the *{infinispan-pod-name}* container with the `--network={podman-network-name}` option: later in this guide, this will allow us to connect to the *{infinispan-pod-name}* container from the *{my-jaxrs-app-infinispan-docker-image-name}* container.
+
+NOTE: you can check the container is running by opening http://0.0.0.0:{infinispan-port}/console/ in your browser and logging in as `{infinispan-user}` / `{infinispan-password}`
+
+== Maven Project
+
+=== web.xml
+
+Add the `<distributable/>` tag to the `src/main/webapp/WEB-INF/web.xml` file (if the `src/main/webapp/WEB-INF/web.xml` doesn't exist then    create it):
+
+.web.xml
+[source,xml]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns="https://jakarta.ee/xml/ns/jakartaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-app_6_0.xsd"
+         version="6.0">
+    <distributable/>
+</web-app>
+----
+
+=== Infinispan Connection
+
+Add a file named `infinispan.cli` to the root of the maven project:
+
+.infinispan.cli
+[source,text,subs="normal"]
+----
+/subsystem=jgroups/channel=ee:write-attribute(name=stack,value=tcp)
+/subsystem=transactions:write-attribute(name=node-identifier,value=${transactions.node-identifier:wildfly1})
+# missing clustering protocol in stack
+/subsystem=jgroups/stack=tcp/protocol=dns.DNS_PING:add(add-index=1, properties={dns_query="_ping._tcp.jgroups-dns-ping.myproject.svc.cluster.local"})
+# connection to the remote infinispan server
+/socket-binding-group=standard-sockets/remote-destination-outbound-socket-binding=remote-infinispan-server:add(host=${infinispan.server.host:0.0.0.0}, port=${infinispan.server.port:{infinispan-port}})
```

Comment:
A default expression value of "0.0.0.0" does not make sense here.

Reply:
removed